pkg/skaffold/schema/latest/config.go

```diff
@@ -612,6 +612,9 @@ type Kustomize struct {
 
 	// BuildArgs are additional args passed to `kustomize build`.
 	BuildArgs []string `yaml:"buildArgs,omitempty"`
+
+	// LifecycleHooks describes a set of lifecycle hooks that are executed before and after every render.
+	LifecycleHooks RenderHooks `yaml:"hooks,omitempty"`
```

Comment:
For an application with k8 manifests only, there is no way to specify the pre-render hooks. Is there a specific reason for it? 
Would it make sense to move it top level to like `Transform` and `Validate` in `RenderConfig`. 
Challenge would be to find a  good config name for it.

Reply:
I agree that this is currently a gap in the code that is here.  My plan was to add a new schema field `rawk8sHooks` and use that as the hooks that run for `rawK8s`.  I also thought of making the hooks be a top level concept but I opted to not to this initially as it would not follow the same design as `deploy`

Reply:
makes sense.